include/oneapi/dpl/pstl/hetero/algorithm_ranges_impl_hetero.h

```diff
@@ -206,6 +266,16 @@ __pattern_any_of(__hetero_tag<_BackendTag>, _ExecutionPolicy&& __exec, _Range&&
         _Predicate{__pred}, oneapi::dpl::__par_backend_hetero::__parallel_or_tag{}, ::std::forward<_Range>(__rng));
 }
 
+template <typename _BackendTag, typename _ExecutionPolicy, typename _R, typename _Proj, typename _Pred>
+bool
+__pattern_any_of(__hetero_tag<_BackendTag> __tag, _ExecutionPolicy&& __exec, _R&& __r, _Pred __pred, _Proj __proj)
+{
+    auto __pred_1 = [__pred, __proj](auto&& __val)
+        { return std::invoke(__pred, std::invoke(__proj, std::forward<decltype(__val)>(__val)));};
+    return oneapi::dpl::__internal::__ranges::__pattern_any_of(__tag, ::std::forward<_ExecutionPolicy>(__exec),
+                oneapi::dpl::views::all_read(::std::forward<_R>(__r)), __pred_1);
```

Comment:
```suggestion
                oneapi::dpl::views::all_read(std::forward<_R>(__r)), __pred_1);
```
In all modified code please remove `::` before `std`. In some places you even have a mix (with and without `::` in one function call).

Reply:
I've changed `::std` to `std` within the new code.